app/industry/[filename]/page.tsx

```diff
@@ -0,0 +1,58 @@
+import { TODAY } from "@/hooks/useFetchEvents";
+import client from "@/tina/client";
+import { Metadata } from "next";
+import IndustryPage from ".";
+import { TinaClient } from "../../tina-client";
+
+type GenerateMetadataProps = {
+  params: { filename: string };
+  searchParams: { [key: string]: string | string[] | undefined };
+};
+
+export async function generateMetadata({
+  params,
+}: GenerateMetadataProps): Promise<Metadata> {
+  const tinaProps = await getData(params.filename);
+  const seo = tinaProps.props.data.industry.seo;
+
+  if (seo && !seo.canonical) {
+    seo.canonical = `${tinaProps.props.data.global.header.url}industry/${params.filename}`;
+  }
+
+  return {
+    ...seo,
+  };
+}
```

Comment:
The seo props for this page aren't being formatted properly so I'd use the hook for that we have. just remember to include the import for useSEO in this file when comitting this suggestion.

```suggestion
  const { seoProps } = useSEO(seo);
  return { ...seoProps };
}
```

Reply:
Done, added changes with new commit